QueAns/admin.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib import admin
+from .models import Question,Meetings,Message,Voting,Satisfaction
+
+
+class MeetingsAdmin(admin.ModelAdmin):
+    def has_change_permission(self, request, obj=None):
+        return (request.method in ['GET', 'HEAD'] and
+                super().has_change_permission(request, obj))
+    def get_readonly_fields(self, request, obj=None):
+        return ['id','venue','moderator','start_time','end_time']
+    def has_add_permission(self, request):
```

Comment:
Please have a line break between function definitions, also PyCharm already shows a warning here, please pay heed to Pycharm warnings.

Reply:
Reformatted the file.